core/new-gui/package.json

```diff
@@ -64,8 +64,8 @@
     "jquery": "~3.6.0",
     "jquery-ui-dist": "~1.13.1",
     "js-abbreviation-number": "~1.4.0",
-    "jszip": "^3.10.1",
     "json-ref-lite": "^1.1.0",
+    "jszip": "^3.10.1",
```

Comment:
is this line needed?

Reply:
jszip was before json. The order was incorrect. yarn add changed the order automatically.

Reply:
ok